atom/common/platform_util_mac.mm

```diff
@@ -252,4 +248,91 @@ void Beep() {
   NSBeep();
 }
 
+#if defined MAS_BUILD
+void SetProcessTitleActivityMonitor(const std::string& name) {
+  // NB: Can't be implemented with public APIs
+}
+#else
+void SetProcessTitleActivityMonitor(const std::string& name) {
+  if (name.length() < 1) {
+    NOTREACHED() << "SetProcessTitleActivityMonitor given bad name.";
+    return;
+  }
+
+  CFStringRef process_name = base::SysUTF8ToCFStringRef(name);
+
+  // Private CFType used in these LaunchServices calls.
+  typedef CFTypeRef PrivateLSASN;
+  typedef PrivateLSASN (*LSGetCurrentApplicationASNType)();
+  typedef OSStatus (*LSSetApplicationInformationItemType)(int, PrivateLSASN,
+                                                          CFStringRef,
+                                                          CFStringRef,
+                                                          CFDictionaryRef*);
+
+  static LSGetCurrentApplicationASNType ls_get_current_application_asn_func =
+      NULL;
```

Comment:
Seems like we use a mixture of `NULL` and `nullptr`, but maybe `s/NULL/nullptr/g` anyway?